jaeseon/app.js

```diff
@@ -0,0 +1,41 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+
+app = express();
```

Comment:
```suggestion
const app = express();
```
app 이라는 변수 명을 const로 선언하여서 추후에 다른 줄에서 의도치 않게 변경되는 상황이 안생기게 해주세요~

Reply:
넵 수정했습니다.